pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "crossfire"
-version = "1.1.0"
+version = "1.2.0"
```

Why does the PR says `1.1.1`, c329cca61980dbf758ac657192051af451085a69 says `1.1.1` but code reads `1.2.0`?

Same code change suggested [before](https://github.com/FelipeSBarros/crossfire/pull/111#discussion_r2638195313):

```suggestion
version = "1.1.1"
```

---

My mistake. Sorry.
I tried to do it interactively, but I didn't realize that it didn't have the expected effect. 
I solved it now with a `git --amend`. 
I hope I didn't do it wrong.
I see that I haven't created another commit, as you suggested, but the previous commit was replaced by a new one.